app/views/groups/show.html.erb

```diff
@@ -1,19 +1,31 @@
 <h1><%= @group.name %></h1>
 <%= @group.description %>
 <br>
+<br>
+<% if @authorized_user %>
+  <%= link_to 'Edit Group', edit_group_path %>
+  <%= link_to 'Delete Group', group_path, method: :delete, data: { confirm: 'Are you sure? This cannot be undone.' } %>
+  <br>
+  <br>
+<% end %>
+<% if !@user_list.empty? %>
+  <%= link_to 'View your Wish List', group_list_path(@group, @user_list[0]) %>
+<% elsif @user_list.empty? && @belonging_user %>
+  <%= link_to 'Create a Wish List', group_lists_path(@group), method: :post %>
+<% end %>
 
+<br>
+<br>
 Users in this group:
 <ul>
   <% @group.users.each do |user| %>
-  <li><%= link_to "#{user.first_name} #{user.last_name}'s profile", profile_path(user.id) %></li>
+  <% wish_list = user.lists.where(['user_id = :user_id and group_id = :group_id', { user_id: user.id, group_id: @group.id }]) %>
```

Comment:
my instinct is to push these query into a method on the user model

Reply:
This is the same query as the method `user_wish_list` now in the group model. I have changed this show page to include that method and move that query logic down.